GoogleScholarPlugin.inc.php

```diff
@@ -113,7 +113,7 @@ public function submissionView($hookName, $args)
         }
 
         // Submission title
-        $templateMgr->addHeader('googleScholarTitle', '<meta name="citation_title" content="' . htmlspecialchars($publication->getLocalizedFullTitle($publicationLocale)) . '"/>');
+        $templateMgr->addHeader('googleScholarTitle', '<meta name="citation_title" content="' . htmlspecialchars($publication->getLocalizedFullTitle($publicationLocale, 'html')) . '"/>');
```

Comment:
Does Google Scholar expect escaped HTML? I would expect this to be plaintext, not HTML.

Reply:
Sorry, my mistake(copy paste code) . It does not . fixed it .